test/e2e/replica_set_readiness_probe/replica_set_readiness_probe_test.go

```diff
@@ -18,6 +18,7 @@ func TestMain(m *testing.M) {
 
 func TestReplicaSetReadinessProbeScaling(t *testing.T) {
 
+	rand.Seed(time.Now().Unix())
```

Comment:
nice, one small thing I would do is to put this in an `init` function

```golang
init(){
    rand.Seed(time.Now().Unix())
}
```

You don't need to explicitly call `init` functions

Reply:
I think init functions are making the code harder to follow as it's very hard to understand when they are executed

see side effect setup in https://medium.com/capital-one-tech/doing-well-by-doing-bad-writing-bad-code-with-go-part-1-2dbb96ce079a

Reply:
@bznein ^